packages/fragment/tests/fragments/dl-2.xml

```diff
@@ -1,11 +1,20 @@
 <FRAGMENT parents="p">
     <dl width="narrow">
-        <li><title>The Title</title>Item 1</li>
```

Comment:
These ones I actually put in on purpose. I believe stuff like this is in the sample article and the contents of list items automagically gets wrapped in `<p>` tags.

Reply:
OK, I'll revert this and perhaps add a `readme.md` to explain why we include non-validating fragment. However, it seems weird to officially support broken PreTeXt...I wonder if it makes more sense to ask Rob to fix this in the sample article?

Reply:
Done!